### ISSUE TITLE
Chlorophyll/Phytoplankton Biomass/NPP

### ISSUE THREAD

**Issue by tylerrohr22** [hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation#3]
1. I've always found it easy to pull data from the OSU site: http://sites.science.oregonstate.edu/ocean.productivity/site.php. They have:
-       MODIS/SEAWIFS/VIIRS Chlorophyll 
-       A Phytoplankton carbon biomass model form backscatter 
-       Several NPP models based on both Chl and Carbon

2. There are, of course, a lot of ways you could use this for model validation. Some obvious starting points
-       Model bias in climatological mean/variance (ie. inter-annual variability) in mean/peak/integrated annual Chl/Carbon/NPP
-       Year to year Co-variability of mean/peak/integrated annual Chl/Carbon/NPP in the model v. Remote sensing record
-       Agreement in phenology (i.e. timing of peak, intitation, and length of bloom)
-       Agreement in biophysical-coupling. Could develop some metrics for the relationship between biological variable and MLD/SST/Eddies which could be compared between model and obs

3. Perhaps a helpful starting point.  Here is the climatogical bias I found for MLD (merged Reanalysis from the OSU sit), NPP (CbPM), and C Biomass (CbPM).  Note: These are from short, 1 degree, offline repeat climatological ocean. So way different than the run in question.  But I suspect the bias distribution may be similar.

![image](https://user-images.githubusercontent.com/119016798/203873426-6d849b49-a6aa-4f88-a79b-5de824f22373.png)

**Comment by hakaseh** [comment]
This is the difference between GPP and NPP in ACCESS-OM2-01-Cycle4:
https://github.com/mom-ocean/MOM5/blob/iamip2-ak/src/mom5/ocean_csiro_bgc/bio_v3.inc#L345-L348

**Comment by matt-csiro** [comment]
Hakase, in line 347 above, it states productivity is being corrected for the the linear mortality term. However, in line 348, grazing (f21) is subtracted from the phytoplankton growth (f11).  
f21 should be f22(?)

**Comment by hakaseh** [comment]
@matt-csiro yes you are correct. f21 should have been f22. thanks for checking. 😭 

npp* in https://github.com/COSIMA/01deg_jra55_iaf/issues/11 does not represent net primary production, but is the gross PP minus grazing (secondary/zooplankton production). In Cycle 4, npp* can be used to derive grazing by subtracting npp* from pprod_gross*. Hopefully npp* can be useful to those interested in zooplankton!

**Comment by tylerrohr22** [comment]
Also would add that most other BGC models I know of don't explicitly resolve autotrophic respiration (AR) but rather include it implicitly in the growth rate term, and then also include a linear mortality term that often is implicitly remineralised by implicit bacteria and returned directly to the nutrient pool. 

My understanding was that this was the case in WOMBAT too. That is, AR was accounted for in the growth rate rather than mortality rate parameter, which would mean NPP=pprod_gross, and not NPP=pprod_gross-linear mortaltily(f21). But I could be wrong, @matt-csiro? I suppose it largely depends on when parameters were tuned if pprod_gross or pprod_gross-f21 was what got evaluated against observed NPP.

Attached below is a schematic of my understanding of NPZD core of WOMBAT. Note, this w/o iron turned on UNLIKE the run we're looking at now, but may be useful:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/119016798/206959077-72c0f159-a322-4036-ad8d-1efa38ac7de5.png">

**Comment by matt-csiro** [comment]
Yes @tylerrohr22, thanks for clarifying and adding details. 
In 'tuning' the ESM, it was pprod_gross that was compared to observed productivity fields/estimates.

**Comment by matt-csiro** [comment]
Attached here are versions of figures from the ACCESS-ESM1.5 description/evaluation paper (Ziehn et al. 2020) with productivity, nutrients and carbon flux, using output from both 1.0 (left) and 0.1 degree (right) ACCESS-OM2-BGC experiments.  
Figure 15 from the ESM1.5 paper is included for reference.  
Time series of global productivity and carbon flux with ACCESS-OM2 experiments are also shown.  

After a quick look, 
the biological pump, or productivity, is on the low side in these experiments, which will affect some regions/metrics.  
For reference, global productivity in the ESM1.5  was 33.5 PgC/y.  
Interannual variability in the global productivity between the two resolutions match well.  
The natural carbon flux is close to zero or balance, and the spatial distribution looks OK.  
The difference between the total and natural carbon fluxes are reasonable with other experiments and the Global Carbon project.  

![append](https://user-images.githubusercontent.com/42162322/207199206-53ec5bb6-c608-445f-90dc-d87e00abbf4b.gif)

![image](https://user-images.githubusercontent.com/42162322/207198570-54663ebb-bd2d-47b5-8c08-568bd13bf53a.png)
 
![prod](https://user-images.githubusercontent.com/42162322/207199412-e88f819e-5055-479c-a828-7b84bddce259.gif)
![carbon](https://user-images.githubusercontent.com/42162322/207199824-5e0469c6-9e02-4d39-a8ab-a4cd7c5d8170.gif)

**Comment by tylerrohr22** [comment]
Here's the comparison of Climatologic NPP and Surface Phytoplankton Biomass. Short story: WOMBAT performs best at high latitudes, particularly the SO, but pretty poorly between 40S and 40N. NPP South of 40S is particularly good compared to CbPM (Fig 2). Overall though both NPP and Biomass appear quite low (Fig 1, 3). But the seasonal cyle is pretty well correlated at high latitudes (Fig 4).  

@matt-csiro This is all converted from Assumed units of N (*106/16), not P (*106), as globall NPP of ~300 PgC seemed far too high and it is documented as mmolN in the saved metadata. Does that seem right to you? To me is seems like the macronutrient is qualitatively P (no nitrogen fixing) but still quantitatively N (i.e parameters optimised against N) 


[ACCESS_OM2_WOMBAT_Validation.pdf](https://github.com/hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation/files/10333824/ACCESS_OM2_WOMBAT_Validation.pdf)

**Comment by tylerrohr22** [comment]
Same figures as in PDF above, just copied here as well for convenience:

![image](https://user-images.githubusercontent.com/119016798/210286516-1e73d273-6996-4ad8-96ac-2d89cdbaa2c9.png)

![image](https://user-images.githubusercontent.com/119016798/210286496-e746985b-a240-4e79-b2b3-79e9d87a4979.png)

![image](https://user-images.githubusercontent.com/119016798/210286533-efb425f7-9615-4fde-a1ef-66f6c16a1aa2.png)

![image](https://user-images.githubusercontent.com/119016798/210286542-d2d89a79-0ace-48ea-afb2-509ebaa05636.png)

**Comment by matt-csiro** [comment]
@tylerrohr22, yes, your interpretation and unit conversions are correct, i.e., qualitatively P though quantitatively N.

**Comment by aekiss** [comment]
@tylerrohr22 how did you calculate NPP for [these plots](https://github.com/hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation/issues/3#issuecomment-1369285170)? Did you use `npp2d` or `pprod_gross_2d` or something else?

**Comment by aekiss** [comment]
Thanks for the [nice diagram](https://github.com/hrsdawson/ACCESS-WOMBAT_01deg_BGC_validation/issues/3#issuecomment-1345845711). I'll just note here for clarity that `01deg_jra55v140_iaf_cycle4` uses somewhat different parameters, that are identical to ACCESS-ESM1.5 as specified in [Ziehn et al. (2020)](https://doi.org/10.1071%2Fes19035), except nat_co2=284.32, which is the value used in the OMIP-BGC protocol (Orr et al. 2017, GMD).
See parameter definitions [here](https://github.com/COSIMA/input_om2-bgc/blob/5ec65b4b6edefde82d2bdb0f8774a12ed9bcde86/interpolate_to_access-om2.ipynb).
